packages/dbs-ai/server/config.js

```diff
@@ -39,6 +40,13 @@ Meteor.startup(function() {
 		i18nLabel: 'DBS_AI_Redlink_Auth_Token'
 	});
 
+	RocketChat.settings.add('DBS_AI_Redlink_Hook_Token', 'key123', {
```

Comment:
@tkurz difference to the setting in https://github.com/mrsimpson/Rocket.Chat/pull/34/files#diff-8e83d6a3163a81272f59768476959fa7 ?